docs/_deployments/ja_aws-lambda.md

```diff
@@ -64,22 +64,16 @@ aws configure
 
 ### Serverless Framework をセットアップする
 
-[Serverless Framework](https://www.serverless.com/open-source/) では、AWS Lambda 向けのアプリの設定、デバッグ、デプロイを簡単に行うためのツールが用意されています。
+[Serverless Framework][serverless-framework] では、AWS Lambda 向けのアプリの設定、デバッグ、デプロイを簡単に行うためのツールが用意されています。
 
 **1. Serverless Framework CLI をインストールする**
 
-Serverless でも [macOS、Windows、Linux](https://www.serverless.com/framework/docs/getting-started/) にインストールして利用できるコマンドラインインターフェイス（CLI）のツールが用意されています。インストールには npm を使用します。次のコマンドを実行してください。
-
-```shell
-npm install --save-dev serverless
-```
-
-> 💡 [Serverless CLI をグローバルにインストールする](https://www.serverless.com/framework/docs/getting-started/)こともできます。その場合は`npm install -g serverless` というコマンドを実行します。
+Serverless でも [macOS、Windows、Linux](https://www.serverless.com/framework/docs/getting-started/) にインストールして利用できるコマンドラインインターフェイス（CLI）のツールが用意されています。インストールするには Serverless の[入門ガイド（英語）](https://www.serverless.com/framework/docs/getting-started/) をお読みください。
```

Comment:
You removed the first link to the getting started page. Can you make this part consistent?